lib/buildingsync/model_articulation/building_section.rb

```diff
@@ -193,10 +193,44 @@ def read_construction_types(section_element, ns)
       end
     end
 
+    def add_principal_hvac_type(buildingSection)
+      # code here
+      buildingSections = buildingSection.parent
```

Comment:
please use snake_case for variable names

Reply:
fixed in a26368e